app/scripts/controllers/network/network-controller.js

```diff
@@ -33,38 +34,58 @@ import { createNetworkClient } from './create-network-client';
  * @property {string} [nickname] - Personalized network name.
  */
 
-const env = process.env.METAMASK_ENV;
-const fetchWithTimeout = getFetchWithTimeout();
+/**
+ * The error code that is used for RPC requests that produce an "internal"
+ * error, borrowed from `eth-rpc-errors`.
+ */
+const INTERNAL_RPC_ERROR_CODE = -32603;
```

Comment:
total nit, but should we add this to `shared/constants/network.ts`?

Reply:
This is also provided as a constant by the `eth-rpc-errors` package, as `errorCodes.rpc.internal`

Reply:
That makes sense! Updated here: [90bc950](https://github.com/MetaMask/metamask-extension/pull/17556/commits/90bc950cd9e05486812b6be7d8d99e48f7e65bbe)